guides/tasks/adding_tasks.md

```diff
@@ -162,9 +162,46 @@ def get_evaluation_scheme_for_task(task) -> BaseEvaluationScheme:
         return SimpleAccuracyEvaluationScheme()
 ```
 
-## 4. Update the supported tasks documentation
+## 4. Add a data downloader
+If your task is publicly available, add the task to the data downloader in [`jiant/scripts/download_data/runscript.py`](../../jiant/scripts/download_data/runscript.py). There are two flavors of downloading tasks:
+
+1) If the task is supported by Hugging Face's [Datasets](https://huggingface.co/nlp/viewer/):
+    - Add the task to `OTHER_HF_DATASETS_TASKS` in [`jiant/scripts/download_data/constants.py`](../../jiant/scripts/download_data/constants.py). Also add the task to `HF_DATASETS_CONVERSION_DICT` in [`jiant/scripts/download_data/dl_datasets/hf_datasets_tasks.py`](../../jiant/scripts/download_data/dl_datasets/hf_datasets_tasks.py). `HF_DATASETS_CONVERSION_DICT` is used to map field changes in Dataset's to the original dataset.
+
+2) If the task is not supported by Hugging Face's Datasets, and the dataset is publicly available for download:
```

Comment:
(i.e. downloadable directly via `wget`, and not behind an authentication wall such as Google Drive)